actors/account/src/lib.rs

```diff
@@ -29,10 +34,12 @@ fil_actors_runtime::wasm_trampoline!(Actor);
 pub enum Method {
     Constructor = METHOD_CONSTRUCTOR,
     PubkeyAddress = 2,
+    AuthenticateMessage = 42,
```

Comment:
@Stebalien I think this is high enough to not conflict with other evm actor methods